lib/buyers/tests/test_api.py

```diff
@@ -75,6 +76,7 @@ def test_get(self):
         eq_(data['pin_is_locked_out'], False)
         eq_(data['pin_was_locked_out'], False)
         eq_(data['email'], self.email)
+        eq_(data['locale'], 'en-US')
```

Comment:
let's leave that up to payments-service, it would be annoying if my data store lower cased things for me :)